Deploy-CAM.ps1

```diff
@@ -1211,11 +1236,17 @@ function New-ConnectionServiceDeployment() {
     $artifactsLocation = $secret.SecretValueText
 
     Write-Host "Using service principal $client in tenant $tenant and subscription $subscriptionId"
-    New-AzureRmRoleAssignment `
-        -RoleDefinitionName Contributor `
-        -ResourceGroupName $csRGName `
-        -ServicePrincipalName $client `
-        -ErrorAction Stop | Out-Null
+    try {
+        New-AzureRmRoleAssignment `
```

Comment:
I think best to change over to the 'check if needed and then assign' method (see other PR) rather than catch the one exception. I wonder what other exceptions we'll hit...